app/templates/gulp/es5/pug.js

```diff
@@ -49,13 +49,13 @@ module.exports = function(gulp, plugins, args, config, taskTarget, browserSync)
     }
 
     return gulp.src([
-      path.join(dirs.source, '**/*.jade'),
+      path.join(dirs.source, '**/*.pug'),
       '!' + path.join(dirs.source, '{**/\_*,**/\_*/**}')
     ])
     .pipe(plugins.changed(dest))
     .pipe(plugins.plumber())
-    .pipe(plugins.jade({
-      jade: jade,
+    .pipe(plugins.pug({
+      pug: pug,
```

Comment:
Looks like the `pug` option is available ([source](https://www.npmjs.com/package/gulp-pug#pugopts)). So feel free to ignore making any changes @benjamin-richardson